readthedocs/search/documents.py

```diff
@@ -81,9 +81,9 @@ def faceted_search(
         }
 
         filters = {}
-        if projects_list:
+        if isinstance(projects_list) and projects_list:
```

Comment:
Doesn't `isinstance` need 2 arguments? What is it checking for an instance of?

Reply:
:man_facepalming: 

You are right.